app/api/v1/endpoints/feedme_endpoints.py

```
@@ -637,51 +616,143 @@
+        FeedMeExample: The updated example.
+    
+    Raises:
+        HTTPException: Returns 404 if the example does not exist, 503 if the FeedMe service is disabled, or 500 on database errors.
+    """
+    
+    if not settings.feedme_enabled:
+        raise HTTPException(status_code=503, detail="FeedMe service is currently disabled")
+    
+    try:
+        with get_db_connection() as conn:
             with conn.cursor(cursor_factory=psycopg2_extras.RealDictCursor) as cur:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)